platforms/documentation/docs/src/snippets/bestPractices/useConventionPlugins/tests/useConventionPlugins-avoid.out

```diff
@@ -0,0 +1,3 @@
+:avoid-this:compileJava SKIPPED
+
+BUILD SUCCESSFUL in 0s
```

Comment:
I think we should try harder to configure something visible in the output here and verify it.

Maybe the convention plugin could wire in a new task that compilation depends upon?

Reply:
+1